src/OrchardCore/OrchardCore.Apis.GraphQL.Abstractions/Extensions/ResolveFieldContextExtensions.cs

```diff
@@ -59,9 +60,9 @@ public static bool HasPopulatedArgument<TSource>(this ResolveFieldContext<TSourc
             return source;
         }
 
-        public static IServiceProvider ResolveServiceProvider<T>(this ResolveFieldContext<T> context)
+        public static IServiceProvider ResolveServiceProvider<T>(this IResolveFieldContext<T> context)
```

Comment:
we should be able to remove this extension.`IResolveFieldContext` now has `RequestServices` as a property